src/components/admin/form-components/ListSelector.jsx

```diff
@@ -41,9 +37,9 @@ export default class ListSelector extends React.Component {
 }
 
 ListSelector.propTypes = {
-  types: PropTypes.object.isRequired,
+  elements: PropTypes.object.isRequired,
   type: PropTypes.string,
   disabled: PropTypes.bool,
   update: PropTypes.func,
-  chosenType: PropTypes.string,
+  chosenElement: PropTypes.string,
```

Comment:
Isn't this required?

Reply:
Or maybe not if we have the default... But then you need to yourself alter the elements to add a none, and make sure that API is clear (that you don't yourself pass in a none in the elements)